codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/testutil/EventStreamTestModels.kt

```diff
@@ -19,6 +19,7 @@ private fun fillInBaseModel(
 ): String = """
     namespace test
 
+    use smithy.framework#ValidationException
```

Comment:
Nit: Prefer making this conditional rather than adding the test dependency to `codegen-client`.

Reply:
- Why is the `ValidationException` not needed in the client?
- I would rather keep the models the same for client/server tests. In a real scenario customers would use the same model for both.

Reply:
> Why is the ValidationException not needed in the client?

Clients don't validate since the server should be the authority on what is valid.